src/components/ui/DataGrid/DataGrid.svelte

```diff
@@ -424,7 +429,14 @@ This has been seen to result in unintended and often glitchy behavior, which oft
   });
 </script>
 
-<div bind:this={gridDiv} class="ag-theme-stellar table" class:highlightOnSelection tabindex="-1" on:focus on:blur />
+<div
+  bind:this={gridDiv}
+  class="ag-theme-stellar table dynamics"
```

Comment:
What is `dynamics` for? Was this to support the row coloring that is no longer used? I see that it is setting `--ag-value-change-value-highlight-background-color` but not sure what the aim is.

Reply:
Good catch - yep this is a remnant of the row coloring we were previously doing but is now gone